package.json

```diff
@@ -45,36 +45,37 @@
     "react-native-start": "packager/packager.sh"
```

Comment:
Yeah -- Babel has this hack where it patches ast-types and I needed to update the dependencies in RN's package.json to get things working.